cli/tests/integrations/test_config.py

```diff
@@ -410,6 +416,9 @@ def test_prepend_url_validation(env):
 
 def test_fail_url_validation(env):
     _fail_url_validation('set', 'core.dcos_url', 'http://bad_domain/', env)
+    _fail_url_validation('set', 'core.dcos_url', 'http://@domain/', env)
+    _fail_url_validation('set', 'core.dcos_url', 'http://a.b:b.a@domain/', env)
+    _fail_url_validation('set', 'core.dcos_url', 'http://user:pass@/', env)
```

Comment:
I think we are going to have a DCOS Image with it soon. @spahl Are we going to be able to enable basic authentication when configuring the vagrant image?